assets/js/index.js

```diff
@@ -1,4 +1,101 @@
 $(document).ready(function(){
+$.fn.isOnScreen = function(){
```

Comment:
This entire file shouldn't be changed. You can return it to how it was with `git checkout assets/js/index.js`